toolbox/tabs/components/ToolbarPanel.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import {Component} from 'react';
+import {hoistComponent} from 'hoist/core';
+
+@hoistComponent()
+export class ToolbarPanel extends Component {
```

Comment:
(I know you were waiting for me to take a look before adding too much content here - just using this as a chance to offer some ideas.)

Will be nice to have a toolbar example with buttons (of course) but also plain-text labels, separators, and some common form fields.